virt/getting_started/virt-using-the-cli-tools.adoc

```diff
@@ -22,7 +22,31 @@ include::modules/virt-installing-virtctl-binary.adoc[leveloffset=+2]
 
 include::modules/virt-installing-virtctl-rhel8-rpm.adoc[leveloffset=+2]
 
-include::modules/virt-virtctl-commands.adoc[leveloffset=+1]
+[id="virtctl-commands_virt-using-the-cli-tools"]
```

Comment:
For the purpose of getting the content ready for conversion, any assembly that has any content anywhere after the first set of includes, with the notable exception of exactly one set of Additional resources that belong to that assembly and not an included module, will ultimately need to be split because said content has nowhere to go in DITA.